src/main/java/com/entropyteam/entropay/employees/services/EmployeeService.java

```diff
@@ -44,4 +54,27 @@ protected Employee toEntity(EmployeeDto entity) {
         employee.setRoles(roles);
         return employee;
     }
+
+    @Override
+    @Transactional
+    public EmployeeDto create(EmployeeDto employeeDto){
+        Employee entityToCreate = toEntity(employeeDto);
+        Employee savedEntity = getRepository().save(entityToCreate);
+        Set<PaymentInformation> paymentInformationSet = paymentInformationService.create(employeeDto.paymentInformation(),savedEntity);
+        savedEntity.setPaymentsInformation(paymentInformationSet);
+        return toDTO(savedEntity);
+    }
+
+    @Override
+    @Transactional
+    public EmployeeDto update(UUID employeeId, EmployeeDto employeeDto) {
+        Employee entityToUpdate = toEntity(employeeDto);
+        entityToUpdate.setId(employeeId);
+        Set<PaymentInformation> paymentInformationSet = paymentInformationService.update(employeeDto.paymentInformation(),employeeId);
+        paymentInformationSet = paymentInformationSet.stream().peek( p -> p.setEmployee(entityToUpdate)).collect(Collectors.toSet());
+        Employee savedEntity = getRepository().save(entityToUpdate);
+        savedEntity.setPaymentsInformation(new HashSet<>(paymentRepository.saveAll(paymentInformationSet)));
```

Comment:
You could remove this line, because we set the payment information in the toDto method